engine/src/main/java/org/terasology/network/internal/NetworkSystemImpl.java

```diff
@@ -756,11 +756,11 @@ public void forceDisconnect(Client client) {
 
     /**
      * Sets the context within which this system should operate
-     * @param context
+     * @param stateContext
```

Comment:
Yes, you are correct. Quite funny though the warning appears since this code is really common. I will revert it to the original and check if it disappears in a newer checkstyle version.

Reply:
The issue here is that either the variable or the method is not correctly named. Checkstyle is therefore not able to detect this as setter method and warns about it.
If the variable should stay as `context` the method should be renamed `setContext`. Of course we could also rename the variable in the `NetworkSystemImpl` to `stateContext`.

Reply:
Can we rename the function to `setContext()`? There is no other reference to "state" in the entire class, so I see no reason to keep the name as `setStateContext()`.

Reply:
Since it was only called in the engine (and tests for the system) it was renamed to `setContext()`